### PR TITLE
First draft of ember-in-viewport post

### PR DESCRIPTION
**Pull request by poteto**
It might be easier to read the non-diff version [here](https://github.com/poteto/reefpoints/blob/lt-ember-in-viewport/source/posts/2015-04-20-ember-in-viewport.md).

### CODE REVIEW

File: source/posts/2015-04-20-ember-in-viewport.md [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+I [wrote a post](https://medium.com/delightful-ui-for-ember-apps/ember-js-detecting-if-a-dom-element-is-in-the-viewport-eafcc77a6f86)
+last year about how I made an Ember Mixin that would let Ember Components or
+Views know if their DOM element had entered or left the viewport. If you're
+unfamiliar with the [`getBoundingClientRect`](https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect)
+API or the approach in general (for determining if an element is in the
+viewport), please have a read of that post first!
+
+This time, I want to talk about how I improved the original Mixin to use the
+[`requestAnimationFrame`](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame)
+API for improved performance at close to 60FPS. Because certain browsers
```
Comment (bcardarella) [inline source/posts/2015-04-20-ember-in-viewport.md]: Might want to link out to: http://caniuse.com/#search=requestAnimationFrame

File: source/posts/2015-04-20-ember-in-viewport.md [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+(mainly IE) don't support `rAF`, we'll also setup an automatic fallback to using
+the Ember run loop method I used in my previous post.
+
+## Demo
+![Featuring Brian](https://d262ilb51hltx0.cloudfront.net/max/1600/1*9WZqJfpL4daIEBJiufTolQ.png)
+
+I made a simple [demo app](http://development.ember-in-viewport-demo.divshot.io/)
+to demonstrate how you might use the Mixin. The goal for this addon was to allow
```
Comment (bcardarella) [inline source/posts/2015-04-20-ember-in-viewport.md]: You might be able to embed within an iframe, we pretty much allow all raw HTML with our markdown

**Comment by bcardarella** [comment]
:+1:

File: source/posts/2015-04-20-ember-in-viewport.md (hunk 2 above)
Comment (poteto) [inline source/posts/2015-04-20-ember-in-viewport.md]: Just tried it out, doesn't look so good. Think I'll stick to the link and image

**Pull request merged**